cluster/gce/gci/configure-helper.sh

```diff
@@ -1455,7 +1455,9 @@ function start-kube-apiserver {
   fi
   if [[ -n "${NUM_NODES:-}" ]]; then
     # If the cluster is large, increase max-requests-inflight limit in apiserver.
-    if [[ "${NUM_NODES}" -ge 1000 ]]; then
+    if [[ "${NUM_NODES}" -ge 3000 ]]; then
```

Comment:
One more thing I earlier forgot - could you please also make such a change for kubemark?

Reply:
done